.babelrc

```diff
@@ -4,15 +4,6 @@
       "@babel/preset-env",
       {
         "modules": false,
-        "targets": {
-          "browsers": [
```

Comment:
AFAIK, `preset-env` doesn't support this config anyway; there shouldn't have been a [`browsers` key](https://babeljs.io/docs/en/babel-preset-env#targets).